geopandas/tests/test_geoseries.py

```diff
@@ -102,9 +102,9 @@ def test_geom_equals_align(self):
 
     def test_align(self):
         a1, a2 = self.a1.align(self.a2)
-        assert a2['A'].is_empty
+        assert not a2['A']
```

Comment:
Can we do `is None` here instead of `not`? I personally find that more explicit, as now my first idea is of it checking for False